### DISCUSSION TITLE
增加敲打空白键次数时，增加显示翻译的种类。以及改进。

### DISCUSSION THREAD

**Discussion by GitHubkaede** [immersive-translate/immersive-translate#1336] [Ideas]
### 新功能 | New Feature

1.键入2次空白键 翻译出 第一语言，键入3次～4次个别都是显示不同语言， 依此类推。 
2.希望改进的小问题：侧边的浮动图标觉得感应像素范围可以低一点，鼠标都还未接触就已经触发弹出。
3.

### 为什么您想要添加这些功能 | Why you want to add these features

1.算是比较快捷的切换方式，不必一直来来回回去设定页面切换。
2.很小的问题，不改也没关系，或许？可以增加范围设置？
3.

**Comment by theowenyoung** [comment]
> 1.键入2次空白键 翻译出 第一语言，键入3次～4次个别都是显示不同语言， 依此类推。

语言可以在输入框开头指定的，比如：

ja 你好

见： https://immersivetranslate.com/docs/input/

>  2.希望改进的小问题：侧边的浮动图标觉得感应像素范围可以低一点，鼠标都还未接触就已经触发弹出。

这个是刻意的，为了更加的自然一点～

**Comment by GitHubkaede** [comment]
> > 1.键入2次空白键 翻译出 第一语言，键入3次～4次个别都是显示不同语言， 依此类推。
> 
> 语言可以在输入框开头指定的，比如：
> 
> ja 你好
> 
> 见： https://immersivetranslate.com/docs/input/請參閱： https://immersivetranslate.com/docs/input/
> 
> > 2.希望改进的小问题：侧边的浮动图标觉得感应像素范围可以低一点，鼠标都还未接触就已经触发弹出。
> 
> 这个是刻意的，为了更加的自然一点～

这个操作有去说明页面看过，
虽然还要打命令有点麻烦，看来只能这样了。

本人刚好比较头脑简单，就比较想直接一点 敲打空白键了事。
不得不说这个功能真的不错。谢谢回应呐。
